src/main.js

```diff
@@ -1,8 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
toto treba ?

Reply:
Jop, lebo inak by sme museli includnut Vue do dependencies. Da sa to ohandlovat , len to chce este nejaky ten research...